wal/src/rocks_impl/manager.rs

```diff
@@ -315,22 +389,38 @@ impl RocksImpl {
                 .context(Decoding)?
             {
                 debug!(
-                    "RocksImpl find table unit pairs stop scanning, because log keys are exhausted"
+                    "RocksImpl find table unit pairs stop scanning by table id, because log keys are exhausted."
                 );
                 break;
             }
 
+            // Found a valid log key by `region_id` and `current_table_id`, check and maybe
+            // insert the related information into `table_max_seqs`.
             let log_key = self
                 .log_encoding
                 .decode_key(iter.key())
                 .map_err(|e| Box::new(e) as _)
                 .context(Decoding)?;
+            debug!(
+                "RocksImpl has found log key by table id, log key:{:?}",
+                log_key
+            );
+
+            if log_key.region_id != region_id {
+                debug!("RocksImpl find table unit pairs stop scanning by table id, because has encountered next region id,
+                    searching table id:{}
+                    current region id:{},
+                    next region id:{}", current_table_id, region_id, log_key.region_id);
```

Comment:
The format for these ids seems weird, how about letting the `cargo fmt` take over the formatting work instead of the manual formatting.